utils/bigint/src/bigint_ser.rs

```diff
@@ -2,18 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use num_bigint::{BigInt, Sign};
-use serde::{Deserialize, Serialize};
+use serde::Serialize;
 
 /// Wrapper for serializing big ints to match filecoin spec. Serializes as bytes.
 #[derive(Serialize)]
 #[serde(transparent)]
 pub struct BigIntSer<'a>(#[serde(with = "self")] pub &'a BigInt);
 
-/// Wrapper for deserializing as BigInt from bytes.
-#[derive(Deserialize, Serialize)]
-#[serde(transparent)]
-pub struct BigIntDe(#[serde(with = "self")] pub BigInt);
```

Comment:
The idea was that all these types will probably go eventually, but I'll keep them for now and then we'll decide later

Reply:
totally, that's cool. But I will mention having this type does come in handy for certain cases when you don't have a struct to annotate on a serde derive. Can be very useful for debugging certain things and in tests (not this one specifically yet tho, but could in future). 

I just don't think there is a disadvantage to having available, especially since we are publishing these crates so having this asymetric API is a bit weird.

Reply:
Ah, didn't think about that!